exercises/strings/countLetter/countLetter.js

```diff
@@ -23,11 +23,23 @@
 
 function countLetter(string, letter) {
   // This is your job. :)
+  var i;
+  var count = 0;
+  for (i=0; i<string.length; i++){
+    if(string.charAt(i) == letter){
```

Comment:
Avoid `==` in JavaScript if you can. It has surprising behavior. For example, `1234 == '1234'` evaluates to `true`. Instead use `===`, which behaves how you'd expect.